python_client/kubetorch/resources/images/images.py

```diff
@@ -8,25 +8,15 @@ def debian() -> Image:
     a minimal Debian base.
     """
     import kubetorch.serving.constants as serving_constants
-    from kubetorch.globals import config
 
-    if config.tracing_enabled:
-        image_id = serving_constants.SERVER_IMAGE_WITH_OTEL
-    else:
-        image_id = serving_constants.SERVER_IMAGE_MINIMAL
-    return Image(name="debian", image_id=image_id)
+    return Image(name="debian", image_id=serving_constants.SERVER_IMAGE_MINIMAL)
```

Comment:
I guess this somehow got lost in a refactor... but we should still support the `_WITH_OTEL` image when `compute._server_should_enable_otel()` is True

Fair to say it's outside the scope of the PR because it's not currently implemented

Reply:
Actually yeah, never implemented! It's only in `compute` if no image is provided